### PR TITLE
Add note about where to find up to date info

### PR DESCRIPTION
**Pull request by cblgh**
I came across this repository in search of federated pull request for gitea (https://github.com/go-gitea/gitea/issues/184), and at the outset this project as a whole seems to be dead.

But it isn't! 🌱 

I've updated the readme with @bill-auger's response from https://github.com/forgefed/forgefed/issues/24#issuecomment-461371494

cc @yookoala @Arkanosis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,25 @@ Gogs, Gitea).
...
 
+## Updates
+### ForgeFed Community Forum
+
+In order to consolidate development and community discussions, there is now a **public web forum** on the socialhub.network website. That forum is the home to ActvityPub and several projects that use ActvityPub as their federation protocol; and so it is a natural fit for ForgeFed. Everyone is invited to use that forum instead of this GitHub repo or the mailing list.
+
+Thanks to SocialHub and FeNeAs for making that possible.
+
+Web Forum: https://socialhub.network/c/forgefed
+### ForgeFed in the Fediverse
```
Comment (waldyrious) [inline README.md]: I'd suggest adding a blank line above this heading and the next one.

Comment (cblgh) [reply README.md]: fixed in https://github.com/forgefed/forgefed/pull/26/commits/fe575c02bbec99acf8ba55bf3cbe405b504bc0f2 thanks for the feedback! :3

**Comment by bill-auger** [comment]
the forum information needs updating actually - the socialhub forum has not worked out as well as we had expected - as of last week, there is a new forum on the FENEAS website; which has a primary category for ForgeFed discussions

https://talk.feneas.org/c/forgefed

there is sufficient reason to believe that it will be better-maintained and reliable going forward; and that it would be best to start using that one before there become too many links pointing to the socialhub forum - we expect that the FENEAS will be the stable community forum into the future

i have updated the pinned issue #24 with the new information - the entire "ForgeFed Community Forum" section should be changed in this PR

**Comment by bill-auger** [comment]
there is also an updated README on the notabug repo https://notabug.org/peers/forgefed

ideally this and the notabug repos should be mirrors of each other

**Comment by bill-auger** [comment]
i suppose i should have tried this sooner; but i was able to mirror the notabug repo as a second repo to this 'forgefed' organization - i will try to keep them in sync for now

https://github.com/forgefed/forge-fed

**Comment by cblgh** [comment]
@bill-auger ah that's definitely a start! 🎉 

i'll leave this open in case the person with write access wakes up from their slumber and decides to perform one last action as maintainer :~

**Comment by cblgh** [comment]
@bill-auger might be worth the effort to write in the description field github has that it's a mirror, and link to the notabug repo in the website field

**Comment by bill-auger** [comment]
its not possible - the only thing that anyone else can do on this repo is to moderate the bug tracker - but no work was ever added to this repo, so there will be no bugs filed - and there is a proper discussions forum now; so there is not much to do on this repo at all, unless koala wants to revive it someday

**Comment by cblgh** [comment]
@bill-auger ah sorry, i meant on your [newly created repo](https://github.com/forgefed/forge-fed). surely it should be possible to direct folx to the notabug repository both in the description and website fields there ^_^

**Comment by nightpool** [comment]
looks like this has since moved also?
https://socialhub.network/t/moving-to-talk-feneas-org/629

File: README.md [hunk 2]
```diff
@@ -8,6 +8,28 @@ Gogs, Gitea).
...
 
+## Updates
+
+### ForgeFed Community Forum
+
+In order to consolidate development and community discussions, there is now a **public web forum** on the socialhub.network website. That forum is the home to ActvityPub and several projects that use ActvityPub as their federation protocol; and so it is a natural fit for ForgeFed. Everyone is invited to use that forum instead of this GitHub repo or the mailing list.
+
+Thanks to SocialHub and FeNeAs for making that possible.
+
+**Web Forum**: https://socialhub.network/c/forgefed
```
Comment (techknowlogick) [inline README.md]:
```suggestion
**Web Forum**: https://talk.feneas.org/c/forgefed
```

**Comment by bill-auger** [comment]
techknowlogick -

that one change is not enough - the preceding paragraph also
mentions socialhub as the forum host

i have updated the pinned issue #24 with the new information -
the entire "ForgeFed Community Forum" section should be changed
in this PR

**Comment by JeffCarpenter** [comment]
@bill-auger not sure if by "wipe" you mean delete or overwrite, but this repo has quite a few stars & watchers which could help with getting more people onboard.

**Comment by bill-auger** [comment]
just a confusion of terminology - "this" thing that has "stars"
and "watchers" is not a "repo" -  it is a forge website (a
project management tool), which is used by some, but not all,
contributors to the over-all forge-fed project - there is much
more to a forge than a VCS repo - a forge may include a git
repo, but also several other optional things that are orthogonal
to version control, such as a wiki, this issue tracker, mailing
lists, forums, and other bells and whistles

the actual "repo" containing contributions to the forge-fed
project is currently mirrored on three different forges: github,
pagure, and notabug (and on several of the contributors local
machines) - currently, the only project management tool that can
be easily shared across different forges is the VCS repo; but
that is exactly the problem that forge-fed aims to solve

**Comment by JeffCarpenter** [comment]
What would it take to get said person to transfer ownership of the repo and/or org?

**Comment by bill-auger** [comment]
there is no ownership of an organization - that is exactly
what distinguishes them from repos under per-user namespaces -
organization have a permissions system which determines which
people can perform certain actions, such as write access to
certain repos

what it would take, is about 20 seconds and 3 mouse clicks, if
that person wanted to do it - but AFAIK, he has not read or
interacted with this bug tracker or the repo in over a year

FWIW, the current repo is mirrored on several different servers
now, including another repo associated with this github
organization, and there is a proper web forum for discussions
now, and also a mastodon presence; so there is not such urgency
to maintain this repo as there was last year

**Comment by rugk** [comment]
Uggh, what happened to this repo now?

Why was not this PR merged?

Can anyone clean up the confusion.

---

I just came in here thorough links (gitea) and only see troubles in this PR, but really don't want to read it all. Mark this as deprecated, if you have a new repo??

**Comment by bill-auger** [comment]
rugk -

please do read threads before posting - your questions have been
asked and answered already - you should not expect anyone to
answer them again only for your benefit - that is common
netiquette

**Comment by rugk** [comment]
This is a huge thread and I've asked for a small summary…

Also there are multiple threads/issues in here already and this does not really help anyone, if one has to dig through a whole repo to find out what's up here…

**Comment by bill-auger** [comment]
yes this repo fell into an unfortunate state of neglect - that
is why there is a single "pinned issue" explaining the current
status - it is the most that can be done with this repo

https://github.com/forgefed/forgefed/issues/24

**Review by nightpool — DISMISSED** [review]

**Comment by rugk** [comment]
> Merging can be performed automatically with 1 approving review.

AFAIK you just need to review it. But **approving**! Not only comment…

**Comment by nightpool** [comment]
@rugk I approved it, and it showed me a different message saying that only an admin can merge pull requests.

**Review by nightpool — DISMISSED** [review]

**Comment by nightpool** [comment]
image

**Review by lafriks — APPROVED** [review]

**Pull request closed without merging**